### ISSUE TITLE
Programmatic usage

### ISSUE THREAD

**Issue by numfin** [basro/stylance-rs#3]
Could you make stylance-cli functionality into library so i can put it inside `build.rs` ?
This way i can use any version (even from git) of stylance generator functionality without installing binary on system

**Comment by ActuallyHappening** [comment]
Yeah, simply publishing a second crate, probably `stylance-cli`, with a `lib.rs` that exports the `run` function is enough for me. I'm not even using it in `build.rs`, just a `cargo xtask` custom script

**Comment by numfin** [comment]
@basro im not sure im going to use rust frontend in near future because of ecosystem immaturity, but its really nice you managed to do lib extraction!!! Glad to see development is happening <3
